src/input.ts

```diff
@@ -6,13 +6,15 @@ export interface Input {
   continuousLabel: string | null
   triggerLabels: string[]
   workflow: string
+  failedJobsOnly: boolean
 }
 
 export function get(): Input {
   const token = core.getInput('token', { required: true })
   const onceLabel = core.getInput('once-label') || null
   const continuousLabel = core.getInput('continuous-label') || null
   let triggerLabels = core.getInput('trigger-labels').split(',')
+  const failedJobsOnly = core.getInput('failed-jobs-only') === 'true' || false
```

Comment:
```suggestion
  const failedJobsOnly = core.getInput('failed-jobs-only') === 'true'
```

should already be a boolean.